src/ui/public/notify/notifier.js

```diff
@@ -39,22 +34,61 @@ function closeNotif(cb, key) {
     // this === notif
     var i = notifs.indexOf(this);
     if (i !== -1) notifs.splice(i, 1);
-    if (this.timerId) this.timerId = clearTO(this.timerId);
+    if (this.timerId) {
+      Notifier.config.clearInterval(this.timerId);
+      this.timerId = undefined;
+    }
     if (typeof cb === 'function') cb(key);
   };
 }
 
+function cancelNotifTimer(cb, key) {
```

Comment:
Since this is a thunk rather than a function that actually executes the cancel-timer action, perhaps a better name may be something like `timerCanceler`.